workspace-manager-clienttests/src/main/java/scripts/testscripts/ServiceStatus.java

```diff
@@ -9,11 +9,28 @@
 import org.slf4j.LoggerFactory;
 import scripts.utils.ClientTestUtils;
 
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
 public class ServiceStatus extends TestScript {
     private static final Logger logger = LoggerFactory.getLogger(ServiceStatus.class);
+    private Duration delay = Duration.ZERO;
+
+    @Override
+    public void setParameters(List<String> parameters) {
+
+        if (parameters == null || parameters.size() == 0) {
+            return;
+        }
+        delay = Duration.ofSeconds(Long.parseLong(parameters.get(0)));
+    }
 
     @Override
     public void userJourney(TestUserSpecification testUser) throws Exception {
+        if (delay.getSeconds() > 0) TimeUnit.SECONDS.sleep(delay.getSeconds());
+
+        logger.info("Starting test script");
```

Comment:
nit: test script has already started, perhaps "Checking status endpoint now"?

Reply:
Updated comment to "Checking status endpoint now".